Server/JokesOnYou.Web.Api/Profiles/TagProfile.cs

```diff
@@ -13,6 +13,7 @@ public class TagProfile : Profile
         public TagProfile()
         {
             CreateMap<Tag, TagReplyDto>();
+            CreateMap<TagCreateDto, Tag>();
```

Comment:
A tag has a OwnerId, how do we make sure that it's correctly set since the Dto doesn't contain it? (Which I think is correct. See: https://github.com/Code2Gether-Discord/JokesOnYou/issues/142 for reference.) 

I guess the person creating the tag is also the owner?

Reply:
Maybe we can set the OwnerId using an Identity field: private readonly UserManager<IdentityUser> _userManager;

Reply:
The Controller already has a way of getting the userId through the Token.
when inside the controller type: `string UserId = ClaimsPrincipalExtension.GetUserId(User);`
That will retrieve the UserId using the token.

Reply:
so a few options.
1. Create a new Dto for the service?
2. add parameters to the service.
anything else ?

Reply:
3. Make the controller responsible for going from Dto to actual model and pass the model on to the service. 
4. Some CQRS implementation using MediatR (never used it, only saw some videos on it), so I'm not even sure if it will solve the issue.

My preference currently: option 2. But I'd go for the solution provided by a more experienced dev :)